frontend/src/Admin/components/ShowSubmissionDetails.js

```diff
@@ -32,6 +32,11 @@ const renderSubmissionByType = submission => {
             alt={submission.title}
             src={`${STATIC_PATH}${getImageThumbnail(submission.path)}`}
           />
+          <div className='text-left'>
+            <a href={`${STATIC_PATH}${submission.path}`} target='_blank'>
+              <Button color='primary'>View Image</Button>
+            </a>
+          </div>
```

Comment:
The button was originally done with an `submission.entryType === PHOTO`. I moved it up here so it can be used on different submission types

Reply:
Does it apply to videos? and should the button text change if it's a pdf?

Reply:
Nvm, I took a look at the file structure, that looks handled.